tests/tasks/setup/eks/awscli-cfn-lt.yaml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: awscli-eks-cfn-launch-template
+  namespace: scalability
+spec:
+  description: |
+    Create an EKS CFN stack to output a launch template.
+    This Task can be used to create an EKS CFN stack that outputs a launch template.
+  params:
+    - name: cluster-name
+      description: EKS cluster you want to create CFN stack for.
+    - name: stack-name
+      description: Stack name you want to spin.
+    - name: region
+      default: "us-west-2"
+      description: The region where the cluster is in.
+    - name: kubernetes-version
+      default: "1.28"
+      description: The EKS version to install.
+    - name: ng-cfn-url
+      description: The url of the CFN YAML/JSON to create CFN stack for NG launch template
+    - name: endpoint
+      default: ""
+    - name: node-max-pod-count
+      default: "35"
```

Comment:
Why 35 ? As long as we use PD, we should be able to allocate more than 35 ?

Reply:
We don't need this anymore